django_project/frontend/urls.py

```diff
@@ -211,6 +212,11 @@
         TotalCountPerActivityAPIView.as_view(),
         name='total_count_per_activity'
     ),
+    path(
+        'species-population-total-density/',
```

Comment:
@ketankartoza can we add 'api/' to the path? e.g. 'api/species-population-total-density/', it's easier to differentiate with a page view and API view.

Reply:
Okay @danangmassandy I will add it in API's